Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
 github "hamchapman/Mockingjay" "90156b01840e39549a451f97fc5b611ecf060d8f"
-github "krzyzanowskim/CryptoSwift" "0.13.0"
+github "krzyzanowskim/CryptoSwift" "0.13.1"
```

Comment:
If there's a change in the `Cartfile.resolved` we should reflect it in the `Podfile` too (i.e. bump the dep to 0.13.1 there too)

Reply:
Where? `PusherChatkit.podspec`? I can't find any other references to `CryptoSwift`

Reply:
Ah yes, my bad. Forgot it was just a dependency for the tests. Ignore me